R/ChromBackend.R

```diff
@@ -0,0 +1,615 @@
+#' @include hidden_aliases.R
+NULL
+
+#' @title Chromatographic data backends
```

Comment:
A general question here given that `ChromBackend` and `MsBackend` share a lot of slots: would it be worth thinking of a way to consolidate these to avoid divergence?

Reply:
I also thought about that. For now I would leave it as it is (i.e. both backends being completely independent). We could make them inherit from the same virtual super-backend later I guess.

Reply:
Actually, I didn't think of a super-class (seems a bit too over-engineering), but rather to either 

- document these *slots shared between different backends* in their own documentation file that would like in one package and the other one would link to it, or 
- a chunk of documentation that we would (ideally automatically) include in the roxygen headers.